tests/TelemetryUtil.ts

```diff
@@ -24,9 +24,6 @@ export const validateTelemetry: (json: string, numPhrases: number, numHypothesis
         expect(hypothesis.length).toEqual(numHypothesis);
     }
 
-    // For review: V2 service does not appear to be sending a PhraseLatencyMs metric, despite sending a 
-    // translation.phrase message
```

Comment:
"service does not appear to be sending a PhraseLatencyMs metric"... Is this something we need to talk to the service folks about?

Reply:
My comment was misinformed, client sends telemetry messages, not service. This PR fixes the lack of PhraseLatencyMs message in telemetry for continuous recognition.